### ISSUE TITLE
Cake builds for every architecture

### ISSUE THREAD

**Issue by bordam** [NickvisionApps/Denaro#518]
Hi, I'm working on updating the aur package. I used the same commands for denaro and tube-converter but while tube-converter behaves correctly, in denaro the build system installs runtimes for every architecture. I'm using these commands:

```
dotnet tool restore
dotnet cake --target=Publish --prefix=/usr --ui=gnome
dotnet cake --target=Install --destdir="$pkgdir"
```

**Comment by nlogozzo** [comment]
The scripts are exactly the same so I don't know why it's happening on one and not the other....
@fsobolev Any ideas?

**Comment by fsobolev** [comment]
I'm not sure what causes this, but the solution is to set runtime ID when publishing, I will commit fix later today.

**Comment by nlogozzo** [comment]
Fixed with V2023.5.0 stable :)
